src/abstractdataframe/show.jl

```diff
@@ -42,7 +42,7 @@ function compacttype(T::Type, maxwidth::Int=8)
         sT = string(T)
         suffix = "⍰"
         # handle the case when after removing Missing union type name is short
-        length(sT) < 8 && return sT * suffix
+        length(sT) ≤ 8 && return sT * suffix
```

Comment:
Shoudn't this use `maxwidth`?

Reply:
Earlier we have agreed on the rule:

> if whole type name fits `maxwidth` then use it otherwise always truncate to 8 to have consistency when printing types.

This is especially important for narrow columns (e.g. when `maxwidth` is 1 or 2)